ui/lib/utils/sentryHelpers.ts

```diff
@@ -0,0 +1,65 @@
+import * as Sentry from '@sentry/react'
+
+const firstMountTransaction = Sentry.startTransaction({ name: 'first mount' })
+const transactionHub = new Map<string, typeof firstMountTransaction>()
+
+export const sentryEnabled = process.env.REACT_APP_SENTRY_ENABLED === 'true'
```

Comment:
Maybe we should control this switch from the backend? same as the current mixpanel?

Reply:
we'll be using that switch to decide if routing and ajax usage should be collected, and should init sentry to collect basic perfs by this env variable.

Reply:
Just curious, `process.env.REACT_APP_SENTRY_ENABLED` shouldn't be boolean type already when we set as `REACT_APP_SENTRY_ENABLED=true` in the env?

Reply:
[according to nodejs doc](https://nodejs.org/api/process.html#process_process_env): 

> Assigning a property on process.env will implicitly convert the value to a string. This behavior is deprecated. Future versions of Node.js may throw an error when the value is not a string, number, or boolean.

dotenv also follows that, there is an issue about this https://github.com/motdotla/dotenv/issues/51